.test-infra/jenkins/metrics_report/report_generator.py

```diff
@@ -27,11 +27,16 @@
 
 INFLUXDB_USER = os.getenv("INFLUXDB_USER")
 INFLUXDB_USER_PASSWORD = os.getenv("INFLUXDB_USER_PASSWORD")
-WORKING_SPACE = os.getenv("WORKSPACE", "")
-PERF_DASHBOARDS = os.path.join(
-    WORKING_SPACE,
-    "src/.test-infra/metrics/grafana/dashboards/perftests_metrics/")
-
+if "GITHUB_RUN_ID" in os.environ:
+  WORKING_SPACE = os.getenv("GITHUB_WORKSPACE", "")
+  PERF_DASHBOARDS = os.path.join(
+      WORKING_SPACE,
+      ".test-infra/metrics/grafana/dashboards/perftests_metrics/")
+else:
+  WORKING_SPACE = os.getenv("WORKSPACE", "")
+  PERF_DASHBOARDS = os.path.join(
+      WORKING_SPACE,
+      "src/.test-infra/metrics/grafana/dashboards/perftests_metrics/")
```

Comment:
```suggestion
# If in GitHub Actions, use the actions workspace, else use the default workspace
WORKING_SPACE = os.getenv("GITHUB_WORKSPACE", os.getenv("WORKSPACE", ""))
PERF_DASHBOARDS = os.path.join(
    WORKING_SPACE,
    "src/.test-infra/metrics/grafana/dashboards/perftests_metrics/")
```

Nit, I think we can simplify this a bit

Reply:
Then we also don't need to pass through `GITHUB_RUN_ID`

Reply:
We still need if because in case of jenkins there is `src` but in case of GHA there isnt one in the path

Reply:
Should we perhaps: 

```suggestion
WORKING_SPACE = os.getenv("GITHUB_WORKSPACE", os.getenv("WORKSPACE", ""))
if "GITHUB_WORKSPACE" in os.environ:
   path_prefix = ""
else:
    path_prefix= ".src/"
PERF_DASHBOARDS = os.path.join(
    WORKING_SPACE,
    path_prefix+".test-infra/metrics/grafana/dashboards/perftests_metrics/")
```

Reply:
Yeah, I think I like this one a bit more, it is clearer what is changing. Thanks